apps/console.py

```diff
@@ -271,7 +272,8 @@ async def run_async(self, device_config, transport):
             if device_config:
                 self.device = Device.from_config_file_with_hci(device_config, hci_source, hci_sink)
             else:
-                self.device = Device.with_hci('Bumble', 'F0:F1:F2:F3:F4:F5', hci_source, hci_sink)
+                random_address = ':'.join(map(lambda c: f"{c:02X}", random.sample(range(255), 6)))
```

Comment:
Is it worth logging the generated address?

Reply:
Would be nice indeed to be able to show which address was picked. The log panel may not be the best output, because it may scroll past that line already when the other log lines are printed at startup. Maybe we could add a ‘device’ tab (Which we could access with `show device`) where we could show that, and maybe other controller info.

Reply:
Added a log. I like the idea of a "device" tab. I'll split that into a different PR.